src/homebridge-roku.js

```diff
@@ -100,29 +102,58 @@ class RokuAccessory {
     return accessoryInfo;
   }
 
+  doesSupportWakeOnLan() {
+    return this.info.supportsWakeOnWlan === 'true';
+  }
+
   setupTelevision() {
     const television = new Service.Television(this.name);
 
     television
       .getCharacteristic(Characteristic.Active)
-      .on('get', (callback) => {
-        this.roku
-          .info()
-          .then((info) => {
-            const value =
-              info.powerMode === 'PowerOn'
-                ? Characteristic.Active.ACTIVE
-                : Characteristic.Active.INACTIVE;
-            callback(null, value);
-          })
-          .catch(callback);
+      .on('get', async (callback) => {
+        try {
+          const info = await pTimeout(this.roku.info(), 1000);
```

Comment:
It looks like you added documentation for a `requestTimeout` param in the homebridge config, all these `pTimeout` calls should be using that variable. Similar to volumeIncrement.